pkg/bpf/bpffs.go

```diff
@@ -129,6 +175,35 @@ func mountFS() error {
 		if err != nil {
 			return fmt.Errorf("Command execution failed: %s\n%s", err, out)
 		}
+	} else { // Already mounted. We need to fail if mounted multiple times.
+
+		// Execute the following command to find multiple bpffs mount points
+		// % mount | grep "<mapRoot> " | wc -l | cut -f1 -d' '
+		newmapRoot := mapRoot + " " // Append space to ignore /bpf/xdp and /bpf/ip mountpoints.
+		mt := exec.Command("mount")
+		grep := exec.Command("grep", newmapRoot)
+		wc := exec.Command("wc", "-l")
+		cut := exec.Command("cut", "-f1", "-d ")
+
+		output, stderr, _ := mountCmdPipe(mt, grep, wc, cut)
+
+		if len(stderr) > 0 {
+			return fmt.Errorf("Command execution failed: %s\n", stderr)
+		}
+		str := string(output)
+
+		//Strip the newline character at the end.
+		parts := strings.Split(string(str), "\n")
```

Comment:
Why not simply `strings.Split(string(output), "\n")`?

Reply:
sure.

Reply:
done